typespec-tests/src/test/java/com/encode/duration/EncodeDurationTests.java

```diff
@@ -31,6 +33,8 @@ public void testQuery() {
         queryClient.int32Seconds(SECOND36);
 
         queryClient.iso8601(DAY40);
+
+//        queryClient.int32SecondsArray(Arrays.asList(SECOND36, Duration.ofSeconds(47)));
```

Comment:
there is bug on this one, will fix in a separate PR (as this one is not about duration).